howso/utilities/utilities.py

```diff
@@ -103,8 +103,6 @@ def trainee_from_df(df, features: Optional[Mapping[str, Mapping]] = None,
             context_features.append(fname)
```

Comment:
The action_features and context_features are no longer used in this method. Should depreacte the method parameter for action_features and remove the collection of context_features.